cranelift-codegen/src/regalloc/coloring.rs

```diff
@@ -1081,6 +1266,11 @@ impl<'a> Context<'a> {
                     regs.global
                         .free(rc, self.cur.func.locations[lv.value].unwrap_reg());
                 }
+            } else {
+                // This is sufficient as-is to pass tests - didn't understand what causes
+                // `process_ghost_kills` and it seems tests might be missing if `RegUnit` being
+                // omitted isn't causing issues here
+                panic!("bring this up in code review");
```

Comment:
I think for completeness this should mimic what's done for `Affinity::Reg`, but I'm not sure how to produce a ghost instruction that would trigger this code path, nor if that not being covered by tests is noteworthy